app/assets/stylesheets/modules/_form.sass

```diff
@@ -1,4 +1,4 @@
 .m-form--row
   margin: $margin 0
   & > *
```

Comment:
`*` selectors should be avoided. any idea why it's there?
the `&` is completely superfluous,, should be removed. (see https://gist.github.com/paulkogel/f9db33d2c06a09dd5539eccf827c53f4)